INFO     root:data_depend.py:37 提取变量：erroranswer: d27353d7-80de-47ad-8b9a-97dbdcda4481
INFO     root:validate.py:76 响应断言:  预期值: '200' 实际值: '200', 断言成功!